content_sync/apis/github.py

```diff
@@ -28,7 +28,9 @@ class GithubApiWrapper:
 
     def __init__(self, website: WebsiteContent):
         """ Initialize the Github API backend for a specific website"""
-        self.git = Github(settings.GIT_TOKEN)
+        self.git = Github(
+            base_url=settings.GIT_API_URL, login_or_token=settings.GIT_TOKEN
```

Comment:
Suggest defaulting `GIT_API_URL` to `None in `settings.py` and then switch this to a conditional kwargs so that we can just let `PyGitHub` default to whatever is appropriate:
```suggestion
            login_or_token=settings.GIT_TOKEN,
            **({"base_url": settings.GIT_API_URL} if settings.GIT_API_URL is not None else {})
```